components/ActionSheet/src/MDCActionSheetController.h

```diff
@@ -143,6 +143,21 @@ __attribute__((objc_subclassing_restricted))
  */
 @property(nonatomic, nonnull, strong) UIColor *backgroundColor;
 
+/**
+ The color applied to the title of the action sheet controller.
+
+ @note If only using a title and the actions have no icons make sure they are different colors so
+ there is a distinction between the title and actions.
+ */
+@property(nonatomic, strong, nullable) UIColor *titleTextColor;
+/**
```

Comment:
Tip: All public APIs need an empty line just before their documentation and just after the declaration. In this case we need to add a newline between lines 152 and 153.

Reply:
Done